### DISCUSSION TITLE
Possibility of labeling runners with the action/workflow/job they picked up

### DISCUSSION THREAD

**Discussion by stbarillas** [actions/actions-runner-controller#2550] [Ideas]
At the GCP metrics explorer level, I'd like to be able to query and visualize how much CPU a specific github workflow uses. In my case a runner can pick up many different workflows. it's not 1:1.

This query does not seem possible at the moment because at the kubernetes level, the runner is not labeled with the workflow job it picks up and runs. Could an option be added where this information gets pushed up?

**Comment by nobbs** [comment]
Someone created an issue for basically exactly this feature request just yesterday, https://github.com/actions/actions-runner-controller/issues/2546 - not sure whether a discussion or an issue is a the better place for this 😅

**Comment by stbarillas** [comment]
Closing and tracking the issue references above instead
